### PR TITLE
Add files via upload

### PR DESCRIPTION
**Pull request by nicolasOrtizS**

### CODE REVIEW

File: Assignment2/NicolásOrtizSuaña.ttl [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+@prefix class: <http://www.ontologies.org/ontologies/class#>
+@prefix computer: <http://www.ontologies.org/ontologies/computer#>
+@prefix user: <http://www.ontologies.org/ontologies/user#>
+@prefix sensor: <http://www.ontologies.org/ontologies/sensor#>
+@prefix measurement: <http://www.ontologies.org/ontologies/measurement#>
+
+:Class01          class:includes :Sensor029;
+                  class:includes :Computer101.
+
+:Measurement8401  measurement:hasTemperature :"29";
```
Comment (ocorcho) [inline Assignment2/NicolásOrtizSuaña.ttl]: Add datatypes here and in the following line

**Comment by ocorcho** [comment]
Are you going to review the changes that I proposed?

**Comment by nicolasOrtizS** [comment]
I change the failures in the file.

File: Assignment2/NicolásOrtizSuaña.ttl [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
++@prefix class: <http://www.ontologies.org/ontologies/class#>
++@prefix computer: <http://www.ontologies.org/ontologies/computer#>
++@prefix user: <http://www.ontologies.org/ontologies/user#>
++@prefix sensor: <http://www.ontologies.org/ontologies/sensor#>
++@prefix measurement: <http://www.ontologies.org/ontologies/measurement#>
++
++:Class01          class:includes :Sensor029;
++                  class:includes :Computer101.
++
++:Measurement8401  measurement:hasTemperature :"29"^^xsd:integer;
```
Comment (ocorcho) [inline Assignment2/NicolásOrtizSuaña.ttl]: It does not make sense to have ":" before the RDF literals. This is an error that also appeared in other files from other students, and I already recommended to remove those. Same in other lines

File: Assignment2/NicolásOrtizSuaña.ttl [hunk 3]
```diff
@@ -8,11 +8,11 @@
 +:Class01          class:includes :Sensor029;
 +                  class:includes :Computer101.
 +
-+:Measurement8401  measurement:hasTemperature :"29"^^xsd:integer;
-+                  measurement:atTime :"2010-06-12T 12:00:12"^^xsd:dateTime.
++:Measurement8401  measurement:hasTemperature "29"^^xsd integer;
```
Comment (ocorcho) [inline Assignment2/NicolásOrtizSuaña.ttl]: This is still wrong. Please check what other students have been doing in the assignments that I have already accepted.

**Pull request closed without merging**